specification/automation/resource-manager/Microsoft.Automation/common/v5/types.json

```diff
@@ -0,0 +1,721 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "4.0",
+    "title": "Common types"
+  },
+  "paths": {},
+  "definitions": {
```

Comment:
Factoringwise, I'm wondering why add this instead of just referencing the usual shared common-types v5 definitions directly?
Are these just an exact copy or are there interesting differences?